src/CommunityToolkit.Aspire.SurrealDb/AspireSurrealDbExtensions.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire;
+using CommunityToolkit.Aspire.SurrealDb;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using SurrealDb.Net;
+
+namespace Microsoft.Extensions.Hosting;
+
+/// <summary>
+/// Provides extension methods for registering SurrealDB-related services in an <see cref="IHostApplicationBuilder"/>.
+/// </summary>
+public static class AspireSurrealDbExtensions
+{
+    private const string DefaultConfigSectionName = "Aspire:Surreal:Client";
+    
+    /// <summary>
+    /// Registers <see cref="SurrealDbClient" /> in the services provided by the <paramref name="builder"/>.
+    /// </summary>
+    /// <param name="builder">The <see cref="IHostApplicationBuilder" /> to read config from and add services to.</param>
+    /// <param name="connectionName">The connection name to use to find a connection string.</param>
+    /// <param name="configureSettings">An optional method that can be used for customizing the <see cref="SurrealDbClientSettings"/>. It's invoked after the settings are read from the configuration.</param>
+    /// <remarks>Reads the configuration from "Aspire:Surreal:Client" section.</remarks>
+    /// <exception cref="InvalidOperationException">If required ConnectionString is not provided in configuration section</exception>
+    public static void AddSurrealClient(
```

Comment:
Similar to the comment regarding the config section, if the type you're registering is `SurrealDbClient` would it make sense for the methods to be `AddSurrealDbClient`?

Reply:
We chosed to remove the `db` prefix when it is not necessary. So, for example, the method to inject a `SurrealDbClient` via DI is called `AddSurreal`.